src/sidebar/components/HypothesisApp.js

```diff
@@ -145,6 +146,17 @@ function HypothesisApp({ auth, frameSync, settings, session, toastMessenger }) {
       data-testid="hypothesis-app"
       style={backgroundStyle}
     >
+      {route === 'notebook' && (
+        <div className="absolute right-0 top-0 m-3">
+          <IconButton
+            title="Close the Notebook"
+            onClick={() => {}}
```

Comment:
What I'm trying to do is make this `onClick` trigger an `eventEmitter.publish('closeNotebook')` somehow.